src/main/java/de/cronn/jira/sync/domain/JiraNamedResource.java

```diff
@@ -0,0 +1,61 @@
+package de.cronn.jira.sync.domain;
+
+import java.util.Objects;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+public abstract class JiraNamedResource extends JiraIdResource implements JiraNamedBean {
+
+	private String name;
+
+	public JiraNamedResource() {
+	}
+
+	public JiraNamedResource(String id, String name) {
+		super(id);
+		this.name = name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	@Override
+	public String getName() {
+		return name;
+	}
+
+	@Override
+	public String toString() {
+		return new ToStringBuilder(this, ToStringStyle.SHORT_PREFIX_STYLE)
+			.append("id", getId())
+			.append("name", name)
+			.toString();
+	}
+
+	@Override
+	public final boolean equals(Object o) {
+		if (this == o) {
+			return true;
+		}
+
+		if (!(o instanceof JiraNamedResource)) {
+			return false;
+		}
+
+		JiraNamedResource that = (JiraNamedResource) o;
+
+		return new EqualsBuilder()
```

Comment:
please use plain `Objects.equals(…)` as it was implemented before.

Reply:
This kind of `equals` was previously implemented in
* `JiraIssueStatus`
* `JiraResolution`
* `JiraVersion`

For `JiraIssueStatus`, for example, it was implemented in [0674aef5](https://github.com/cronn-de/jira-sync/commit/0674aef5). Do you remember why? And should I keep the implementation for the three classes above?

Reply:
To be discussed and changed as follow-up.